src/lib/pages/frontsite/Documents/DocumentDetails/LoanRequestForm/LoanRequestForm.js

```diff
@@ -99,7 +100,8 @@ class LoanRequestForm extends Component {
     return (
       <Form.Field>
         <label>Do you require it before a certain date? (optional)</label>
-        <DatePicker
+        <LocationDatePicker
+          locationPid={sessionManager.user.locationPid}
```

Comment:
what happens here when user is logged out?

Reply:
If the user is logged out, this component `LoanRequestForm` is not rendered, instead it displays the `Sign in to loan`
![image](https://user-images.githubusercontent.com/15194802/94433417-94522400-0198-11eb-9381-1125b979dd6c.png)